content/source/docs/enterprise/api/ssh-keys.html.md

```diff
@@ -151,16 +183,34 @@ curl \
 
 ## Update an SSH Key
 
-Update an SSH Key.
+`PATCH /ssh-keys/:ssh_key_id`
 
-| Method | Path           |
-| :----- | :------------- |
-| PATCH | /ssh-keys/:ssh_key_id |
+Parameter            | Description
+---------------------|------------
+`:ssh_key_id`        | The SSH key ID to update.
 
-### Parameters
+This endpoint lets you replace the name or value of an existing SSH key. Existing workspaces that use the key will be updated with the new name and/or key text.
 
-- `:ssh_key_id` (`string: <required>`) - specifies the SSH key ID to update
-- `name` (`string: <required>`) - specifies the name of the SSH key
+Only members of the owners team (or the owners team's service account) can edit SSH keys.
```

Comment:
The use of `'s` with the team/org service accounts is inconsistent in this PR. I think I prefer without:

the organization service account
the owners team service account